routes/root.js

```diff
@@ -322,6 +324,10 @@ router.get('/search/:id', (req, res) => {
   let page = parseInt(rawSplit[1], 10);
   let query = {};
   const terms = [];
+
+  function regexEscape(input) {
```

Comment:
Could this be named something a little more descriptive? Is it escaping regexes?